certified-connectors/Kanbanize/apiDefinition.swagger.json

```diff
@@ -0,0 +1,910 @@
+                    "operationId": "GetTypes",
+                    "value-path": "name",
+                    "value-collection": "data",
+                    "value-title": "name"
+                  }
+                },
+                "template": {
+                  "type": "string",
+                  "description": "template"
+                },
+                "column": {
+                  "type": "string",
+                  "description": "column",
+                  "title": "",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetColumns",
+                    "value-path": "name",
+                    "value-collection": "data",
+                    "value-title": "name",
+                    "parameters": {
+                      "board_id": {
+                        "parameter": "boardid"
+                      }
+                    }
+                  },
+                  "x-ms-dynamic-list": {
+                    "operationId": "GetColumns",
+                    "itemsPath": "data",
+                    "itemValuePath": "name",
+                    "itemTitlePath": "name",
+                    "parameters": {
+                      "board_id": {
+                        "parameterReference": "body/boardid"
+                      }
+                    }
+                  }
+                },
+                "lane": {
+                  "type": "string",
+                  "description": "lane",
+                  "title": "",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetLanes",
+                    "value-path": "name",
+                    "value-collection": "data",
+                    "value-title": "name",
+                    "parameters": {
+                      "board_id": {
+                        "parameter": "boardid"
+                      }
+                    }
+                  },
+                  "x-ms-dynamic-list": {
+                    "operationId": "GetLanes",
+                    "itemsPath": "data",
+                    "itemValuePath": "name",
+                    "itemTitlePath": "name",
+                    "parameters": {
+                      "board_id": {
+                        "parameterReference": "body/boardid"
+                      }
+                    }
+                  }
+                },
+                "position": {
+                  "type": "string",
+                  "description": "position"
+                },
+                "exceedingreason": {
+                  "type": "string",
+                  "description": "exceedingreason"
+                }
+              },
+              "required": [
+                "boardid"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/index.php/api/kanbanize/add_subtask/format/json": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
@dimitar-kanbanize Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.